addons/website_sale/static/src/js/systray_items/product_product_add_form.js

```diff
@@ -1,27 +1,20 @@
 /** @odoo-module **/
 
-import {NewContentFormController} from '@website/js/new_content_form';
-import viewRegistry from 'web.view_registry';
-import FormView from 'web.FormView';
+import {NewContentFormController, NewContentFormView} from '@website/js/new_content_form';
+import {registry} from "@web/core/registry";
 
-const AddProductFormController = NewContentFormController.extend({
+export class AddProductFormController extends NewContentFormController {
     /**
      * @override
      */
-    _getPath(state) {
-        return state.data.website_url.split('#')[0];
-    },
-});
+    get path() {
+        return this.model.root.data.website_url.split('#')[0];
```

Comment:
Is that actually needed ? Would be nice to find the commit that introduced that and add the why as a documentation comment

Reply:
No, I think this is related to a "non-adapted" version of website-backend code where `'?enable_editor=1'` was added to the target url... I removed this now since `website_url` is OK for the action. Thanks :+1: